packages/hash/frontend/src/pages/index.page.tsx

```diff
@@ -1,44 +1,23 @@
 import { useEffect } from "react";
 import { useRouter } from "next/router";
 
-import { Box, Container, Typography } from "@mui/material";
-import { useLogoutFlow } from "../components/hooks/useLogoutFlow";
+import { Container, Typography } from "@mui/material";
 import { useLoggedInUser } from "../components/hooks/useAuthenticatedUser";
 import { NextPageWithLayout } from "../shared/layout";
-import { Button } from "../shared/ui";
 
 const Page: NextPageWithLayout = () => {
   const router = useRouter();
-  const { authenticatedUser, kratosSession } = useLoggedInUser();
-
-  const { logout } = useLogoutFlow();
+  const { authenticatedUser, loading, kratosSession } = useLoggedInUser();
 
   useEffect(() => {
     if (authenticatedUser) {
       void router.push(`/${authenticatedUser.userAccountId}`);
     }
-  }, [router, authenticatedUser]);
+  }, [router, authenticatedUser, kratosSession, loading]);
 
-  /** @todo: remove session developer information */
   return (
     <Container sx={{ pt: 10 }}>
-      {authenticatedUser && (
-        <Typography variant="h1">
-          Hi {authenticatedUser.preferredName}!
-        </Typography>
-      )}
-      <Typography variant="h2" gutterBottom>
-        {kratosSession
-          ? "You have a kratos session"
-          : "You don't have a kratos session"}
-      </Typography>
-      <Button onClick={logout}>Log out</Button>
-      <Typography gutterBottom>
-        This is what your kratos session looks like:
-      </Typography>
-      {kratosSession ? (
-        <Box component="pre">{JSON.stringify(kratosSession, null, 2)}</Box>
-      ) : null}
+      <Typography variant="h1">Homepage</Typography>
```

Comment:
My understanding is this would be a bug to get stuck on. Will user's see it in between React renders? 

If so, I think just seeing a Homepage H1 tag might not be a great experience, perhaps we could use a `Suspense` or something component?

Reply:
> Will user's see it in between React renders?

Yes, when they go to the homepage (because the redirect happens client side).

> If so, I think just seeing a Homepage H1 tag might not be a great experience, perhaps we could use a Suspense or something component?

What do you mean? When would the suspense component render, and what would it display? If it's going to wait on the result of `authenticatedUser`, then we might as well make this a blank page because users will never see the contents of the suspense component.

I think the ideal solution would be for the redirect to happen server-side, which should be possible - but would require us to introduce a bunch of server-side things that let the server-side code make authenticated requests to the GQL API (for the `me` query), and I'm not sure that's worth doing right now. Especially given that I assume we will eventually have a homepage with things to show to logged out users (correct me if I'm wrong @nonparibus), and won't just automatically redirect them to the login page.